docs/topics/forms/formsets.txt

```diff
@@ -16,16 +16,26 @@ form::
     ...     pub_date = forms.DateField()
 
 You might want to allow the user to create several articles at once. To create
-a formset out of an ``ArticleForm`` you would do::
+a formset out of an ``ArticleForm`` you need to do two things. First, you need
+to create the formset class, after which you need to create instances of that
+class.
+
+Creating the formset class is different from the usual way of creating classes
+for Forms or Models in that you don't use the ``class`` statement. Django
+provides a function that creates the class for you::
 
     >>> from django.forms import formset_factory
     >>> ArticleFormSet = formset_factory(ArticleForm)
 
-You now have created a formset named ``ArticleFormSet``. The formset gives you
-the ability to iterate over the forms in the formset and display them as you
-would with a regular form::
+You now have a formset class named ``ArticleFormSet`` from which you can create
+instances in your views.
 
```

Comment:
Given that the example follows, I think a smaller change would be enough here: 

```
You now have created a formset class named ``ArticleFormSet``. This gives you
the ability to iterate over the forms in the formset and display them as you
would with a regular form::
```

Then straight into the existing example. (I think the one line of code says more, more clearly, that a whole load of text.)

Reply:
Please bear with me here - I'm not used to reading diffs in line, and I'm not sure I understand your comment.
What I think you're saying is that you don't think it's necessary to move the line "This gives you ... form" after the line ">>>formset = ...".
If that's the case, then I would disagree. As I read the paragraph you highlighted in your comment, the demonstrative pronoun "This" that starts the second sentence is a reference to the creation of the formset class in the previous sentence - and *this* is what I find problematic with this section of the documentation.

If you don't wish to split that subsequent example, then perhaps that text could be written as:
```
You now have created a formset class named ``ArticleFormSet``. 
Creating a formset gives you the ability to iterate over the forms in the formset and display them as you
would with a regular form::
```
And this would allow that example to remain as-is.

Reply:
How about then, the second sentence beginning... 

```
Instantiating the formset gives...
```

Reply:
I like! Concise and to the point. 

To not fill this thread with needless information, is there a guide you can point me to describing the appropriate process for me to make this modification?